core/src/main/java/com/linecorp/armeria/client/ResponseAs.java

```diff
@@ -105,6 +105,22 @@ static <T> FutureResponseAs<ResponseEntity<T>> json(Class<? extends T> clazz) {
         return aggregateAndConvert(AggregatedResponseAs.json(clazz));
     }
 
+    @UnstableApi
+    static <T> FutureResponseAs<ResponseEntity<T>> json(Class<? extends T> clazz,
+                                                        HttpStatusPredicate predicate) {
```

Comment:
Global comment:
We cannot expose `HttpStatusPredicate` and `HttpStatusClassPredicate` to the public API because it's package private classes.
Instead, could you add method variants that take `HttpStatus`, `HttpStatusClass` and `Predicate<HttpStatus>`, please?

Reply:
I'm sorry, I was careless.
I'll address your comments quickly.
Thank you for your comments!